AP Chem/Unit 7.html

```diff
@@ -36,7 +36,7 @@ <h5>The Equilibrium Constant:</h5>
             <equation>k<sub>eq</sub> &gt; 1</equation>, the reaction favors the forwards direction since there is a higher concentration of products.</li>
         <li>Reversible reactions at equilibrium with different states of matter are said to be in <u>Heterogeneous Equilibrium</u>. In these cases we ignore the solvents and the solids because they are in excess, so it makes sense to not take into account
             such high concentrations into our formula for the equilibrium constant.</li>
-        <li>In equilibrium reactions, pay attention to the state of matter. <strong style="color:mediumvioletred;">Concentrations of solids and liquids are omitted from all calculations</strong>.</li>
+        <li>In equilibrium reactions, pay attention to the state of matter. <strong style="color:mediumvioletred;">Concentrations of solids and liquids are omitted from all calculations</strong>. Another way to think of this is that liquids and solids don't have concentrations to begin with, there is no such thing as stronger water.</li>
```

Comment:
Oh hehe dude I added this line in class when Ms. Cooper was going over it&mdash;just thought you should know